docs/en/glossary/glossary.asciidoc

```diff
@@ -454,6 +474,16 @@ depending on the characteristics of the event. Popular filter plugins include
 grok, mutate, drop, clone, and geoip. Filter stages are optional.
 //Source: Logstash
 
+[[glossary-fleet]] Fleet::
+A web-based UI in {kib} used to add and manage both {agent}s, and integrations for
+popular services and platforms.
+See {fleet-guide}/fleet-overview.html[{agent} overview].
+//Source: Observability
+
+[[glossary-fleet-server]] Fleet Server::
+// tbd
+//Source: Observability
```

Comment:
@dedemorton Can you provide a Fleet Server definition?

Reply:
{fleet-server} is a component to centrally manage {agent}s. It serves as a control plane for updating agent policies, collecting status information, and coordinating actions across actions.

Reply:
Thanks! Updated.